src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -90,3 +91,50 @@ def put(self, request, group, integration_id):
         # TODO(jess): would be helpful to return serialized external issue
         # once we have description, title, etc
         return Response(status=201)
+
+    def post(self, request, group, integration_id):
+        organization_id = group.project.organization_id
+        try:
+            # check org permissions
+            # TODO(jess): should this eventually check ProjectIntegration?
+            integration = OrganizationIntegration.objects.filter(
+                integration_id=integration_id,
+                organization_id=organization_id,
+            ).select_related('integration').get().integration
+        except OrganizationIntegration.DoesNotExist:
+            return Response(status=404)
+
+        if not integration.has_feature(IntegrationFeatures.ISSUE_SYNC):
+            return Response(
+                {'detail': 'This feature is not supported for this integration.'}, status=400)
+
+        installation = integration.get_installation()
+        try:
+            data = installation.create_issue(request.DATA)
+        except IntegrationError as exc:
+            return Response({'detail': exc.message}, status=400)
+
+        external_issue = ExternalIssue.objects.get_or_create(
+            organization_id=organization_id,
+            integration_id=integration.id,
+            key=data['key'],
+            defaults={
+                'title': data.get('title'),
+                'description': data.get('description'),
+            }
+        )[0]
```

Comment:
Why does this return a list?

Reply:
It returns a tuple of `(instance, created (bool))`

Reply:
Ah, I've been doing `item, _ = Thing.objects.get_or_create`